common-libs/url-resolver/package.json

```diff
@@ -29,9 +29,9 @@
     "tsc": "tsc -p tsconfig-build.json",
     "lint": "eslint src test",
     "test": "npm run test:unit && npm run test:integration",
-    "test:unit": "NODE_ENV=test mocha -r ts-node/register test/unit/index.test.ts -p ./tsconfig.json --timeout 8000",
-    "test:integration": "NODE_ENV=test mocha -r ts-node/register test/integration/index.test.ts -p ./tsconfig.json --timeout 10000 --exit",
-    "test:coverage": "NODE_ENV=test nyc --reporter=lcov mocha -r ts-node/register test/unit/index.test.ts -p ./tsconfig.json --timeout 8000",
+    "test:unit": "cross-env NODE_ENV=test mocha -r ts-node/register test/unit/index.test.ts -p ./tsconfig.json --timeout 55000",
```

Comment:
Is 8 seconds is not enough? :D

Reply:
I've just copied commands from other packages in monorepo (my problem with original commands was that they don't run under Windows)

Reply:
It's 55 seconds everywhere in monorepo

Reply:
You could've just added "cross-env". IMO timeout for unit tests should stay extremely low.

Reply:
That's another topic.
Easier to have all test commands in all test packages be the same than to approach every package individually, especially if that's a package I'm not interested at all and only interested in getting the tests to pass.